aws-java-sdk-core/src/test/resources/resources/profileconfig/ProfileWithRole.tst

```diff
@@ -0,0 +1,7 @@
+[source]
```

Comment:
Can you add a test where the source is defined after the role profile? I feel like that would be an easy case to miss if we ever refactored the profile code.